appsec-kit-starter/src/main/java/com/vaadin/appsec/service/AppSecDevToolsPlugin.java

```diff
@@ -43,4 +42,24 @@ public boolean handleMessage(String command, JsonObject data,
         devToolsInterface.send("command-received", data);
         return true;
     }
+
+    private void sendScanResult(int vulnerabilityCount,
+            DevToolsInterface devToolsInterface) {
+        LOGGER.info("Scan completed");
+        sendData("appsec-kit-scan", vulnerabilityCount, devToolsInterface);
+        LOGGER.info(
+                "Vulnerabilities sent to the client: " + vulnerabilityCount);
+    }
+
+    private void refreshScanResult(int vulnerabilityCount,
+            DevToolsInterface devToolsInterface) {
+        sendData("appsec-kit-refresh", vulnerabilityCount, devToolsInterface);
+    }
```

Comment:
This is another one-liner, no need for it.

Reply:
I agree. I can refactor this.